src/test/java/pages/ContextMenuPage.java

```diff
@@ -0,0 +1,63 @@
+package pages;
+
+import org.openqa.selenium.*;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+import javax.swing.*;
+
+import static org.testng.Assert.fail;
+
+public class ContextMenuPage extends BasePage{
+    public static final String ENDPOINT_CONTEXT_MENU = "/context_menu";
+    public static final By BOX_LOCATOR = By.id("hot-spot");
+
+    Actions actions;
+    Alert alert;
```

Comment:
better to create new objects in methods where they're used and not to share them across methods

Reply:
I moved Action to the used method but Alert is used in several methods therefore it's better off to keep it apart. Am I right?

Reply:
Not actually, that is a bad practice and also not 'thread save'